inference-engine/samples/speech_sample/fileutils.cpp

```diff
@@ -0,0 +1,190 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include <iostream>
+
+#include <samples/common.hpp>
+#include <samples/slog.hpp>
+
+#include "fileutils.hpp"
+#include "cnpy/cnpy.h"
+
+
+void ArkFile::GetFileInfo(const char* fileName,
+    uint32_t numArrayToFindSize,
+    uint32_t* ptrNumArrays,
+    uint32_t* ptrNumMemoryBytes) {
+    uint32_t numArrays = 0;
+    uint32_t numMemoryBytes = 0;
+
+    std::ifstream in_file(fileName, std::ios::binary);
+    if (in_file.good()) {
+        while (!in_file.eof()) {
+            std::string line;
+            uint32_t numRows = 0u, numCols = 0u, num_bytes = 0u;
+            std::getline(in_file, line, '\0');  // read variable length name followed by space and NUL
+            std::getline(in_file, line, '\4');  // read "BFM" followed by space and control-D
+            if (line.compare("BFM ") != 0) {
+                break;
+            }
+            in_file.read(reinterpret_cast<char*>(&numRows), sizeof(uint32_t));  // read number of rows
+            std::getline(in_file, line, '\4');                                   // read control-D
+            in_file.read(reinterpret_cast<char*>(&numCols), sizeof(uint32_t));  // read number of columns
+            num_bytes = numRows * numCols * sizeof(float);
+            in_file.seekg(num_bytes, in_file.cur);                               // read data
+
+            if (numArrays == numArrayToFindSize) {
+                numMemoryBytes += num_bytes;
+            }
+            numArrays++;
+        }
+        in_file.close();
+    }
+    else {
+        fprintf(stderr, "Failed to open %s for reading in GetKaldiArkInfo()!\n", fileName);
```

Comment:
update function names in all error messages

Reply:
Or you can use __func__ macro

Reply:
All function names were updated.